prototype/jurisdictions/certificate.AU.xml

```diff
@@ -27,23 +26,12 @@
 					<option value="unsure">
 						<label>you're not sure if you have the rights to publish this data as open data</label>
 					</option>
-					<option value="complicated">
```

Comment:
@Stephen-Gates We shouldn't drop this option for the Australian certificate. It was added because it's genuinely the case that some datasets can have complicated rights associated with them (the main use case was Open Corporates).